uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
+
+#
+# Heater State
+# 
+uint8 FLAG_HEATER_AVAILABLE      = 1
+uint8 FLAG_HEATER_WORKING        = 2
+uint8 FLAG_HEATER_OVERCURRENT    = 4
+uint8 FLAG_HEATER_OPENCIRCUIT    = 8
+uint8 flags
+
+#
+# Pressure Data
+#
+float32 static_pressure                 # Pascal
+float32 differential_pressure           # Pascal
+
+#
+# Temperature Data
+#
+float16 static_p_sensor_temperature     # Kelvin
+float16 diff_p_sensor_temperature       # Kelvin
```

Comment:
I'm not entirely convinced of the advantages of shorter names. They make the code slightly less intelligible, so the policy should be to avoid uncommon abbreviations whenever possible. The additional info given by using a longer name is that it becomes clear that we're talking about a pressure sensor here, and not about a P-sensor, which could mean, say, a proximity sensor. Granted, the reader will be able to understand what does the P mean in this context, but it is better to spare them the extra mental effort whenever possible.